pkg/utils/gadgettracermanagerloglevel/gadgettracermanagerloglevel.go

```diff
@@ -26,7 +26,7 @@ func LogLevel() log.Level {
 				loglevel = level
 				return
 			} else {
-				log.WithError(parseErr).Error(fmt.Sprintf("Invalid log level, valid levels are: %v", strings.Join(strLevels, ", ")))
+				log.WithError(parseErr).Error(fmt.Sprintf("Invalid log level, valid levels are: %v, defaulting to Info", strings.Join(strLevels, ", ")))
```

Comment:
Do we want %s rather than %q otherwise I see the escape characters ``` level=error msg="Invalid log level, valid levels are: panic, fatal, error, warning, info, debug, trace, defaulting to \"info\"" error="not a valid logrus Level: \"xxxx\""``` ?

Reply:
I have 
```shell
level=error msg="Invalid log level, valid levels are: panic, fatal, error, warning, info, debug, trace, defaulting to info" error="not a valid logrus Level: \"xxxx\""
``` 
from 
```go
log.WithError(parseErr).Errorf("Invalid log level, valid levels are: %v, defaulting to %s", strings.Join(strLevels, ", "), log.InfoLevel)
```

Reply:
I am very grateful for all this help. I'm sure I'm quite close to having this near completion.